revocation.go

```diff
@@ -766,7 +766,7 @@ func (rs *RevocationStorage) Load(debug bool, dbtype, connstr string, settings R
 		}
 		if err := rs.sqldb.Delete(IssuanceRecord{}, "valid_until < ?", time.Now().UnixNano()); err != nil {
 			err = errors.WrapPrefix(err, "failed to delete expired issuance records", 0)
-			raven.CaptureError(err, nil)
+			sentry.CaptureException(err)
```

Comment:
irmago does not have its own Sentry handling, so these exceptions will be sent into a black hole now. I think we should do something else here.